### ISSUE TITLE
Error: Failed to find 'minima'

### ISSUE THREAD

**Issue by yazinsai** [mhanberg/jekyll-postcss#23]
Hi there, followed the README to get this setup with a fresh Jekyll project (literally `jekyll new myproject`). I get this error after running `jekyll s`:

```
Configuration file: /Users/rock/projects/qun/jekyll/_config.yml
PostCSS Server listening on port 8124...
            Source: /Users/rock/projects/qun/jekyll
       Destination: /Users/rock/projects/qun/jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
PostCSS Error!

Error: Failed to find 'minima'
  in [
    /Users/rock/projects/qun/jekyll
  ]
```

Minima is the default theme used with Jekyll, but I have no idea why that would be throwing an error. Any tips on how I can get this fixed?

EDIT: Using `jekyll 4.2.0`, ruby 3.0.1p64 (2021-04-05 revision 0fb782ee38) [x86_64-darwin20]

**Comment by Atem18** [comment]
@yazinsai Can you post your configuration files ? I have a working configuration with Tailwind so I can confirm that it works.

**Comment by Agusnez** [comment]
Hello, I have the same issue. I want to set up Jekyll with Tailwind but still using the _minima_ theme. For some reason I cannot import _minima_ like before configuring PostCSS.

@yazinsai Have you made any progress on this?

Thank you.

**Comment by yazinsai** [comment]
@Agusnez yep, got it fixed in my repo here: https://github.com/yazinsai/jekyll-tailwind-template -- should just work

**Comment by wgmyers** [comment]
@yazinsai It looks like you have fixed the issue by removing minima entirely from your example.

Is there a way to fix this issue without removing minima?

It looks like the problem is with postcss-import not knowing where to look for the minima files, but the documentation of postcss-import does not clearly explain how to set the paths that are searched.

Any help would be greatly appreciated.

**Comment by aarishgilani** [comment]
Hey running into same issue here, did anyone figure this out? I want to use tailwind with postcss while still using minima theme.
